modules/installation-guide/examples/system-variables.adoc

```diff
@@ -168,39 +166,39 @@ pass:[<!-- vale off -->]
 ,=== 
  Environment Variable Name,Default value, Description 
  
- `+CHE_WORKSPACE_PLUGIN__BROKER_METADATA_IMAGE+`,"`+quay.io/eclipse/che-plugin-metadata-broker:v3.4.0+`","Docker image of {prod-short} plugin broker app that resolves workspace tooling configuration and copies plugins dependencies to a workspace Note these images are overridden by the {prod-short} Operator by default; changing the images here will not have an effect if {prod-short} is installed via Operator." 
- `+CHE_WORKSPACE_PLUGIN__BROKER_ARTIFACTS_IMAGE+`,"`+quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0+`","Dockerimage of {prod-short} plugin broker app that resolves workspace tooling configuration and copies plugins dependencies to a workspace Note these images are overridden by the {prod-short} Operator by default; changing the images here will not have an effect if {prod-short} is installed via Operator." 
- `+CHE_WORKSPACE_PLUGIN__BROKER_DEFAULT__MERGE__PLUGINS+`,"`+false+`","Configures the default behavior of the plugin brokers when provisioning plugins into a workspace. If set to true, the plugin brokers will attempt to merge plugins when possible (i.e. they run in the same sidecar image and do not have conflicting settings). This value is the default setting used when the devfile does not specify otherwise, via the 'mergePlugins' attribute." 
- `+CHE_WORKSPACE_PLUGIN__BROKER_PULL__POLICY+`,"`+Always+`","Docker image of {prod-short} plugin broker app that resolves workspace tooling configuration and copies plugins dependencies to a workspace" 
+ `+CHE_WORKSPACE_PLUGIN__BROKER_METADATA_IMAGE+`,"`+quay.io/eclipse/che-plugin-metadata-broker:v3.4.0+`","Docker image of {prod-short} plugin broker app that resolves workspace tools configuration and copies plugins dependencies to a workspace. The {prod-short} Operator overrides these images by default. Changing the images here will not have an effect if {prod-short} is installed using the Operator." 
+ `+CHE_WORKSPACE_PLUGIN__BROKER_ARTIFACTS_IMAGE+`,"`+quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0+`","Docker image of {prod-short} plugin artifacts broker. This broker runs as an init container on the workspace Pod. Its job is to take in a list of plugin identifiers (either references to a plugin in the registry or a link to a plugin meta.yaml) and ensure that the correct .vsix and .theia extensions are downloaded into the /plugins directory, for each plugin requested for the workspace." 
```

Comment:
if this is upstream doc only, great. If this is downstreamed, then the `CHE_WORKSPACE_PLUGIN__BROKER_ARTIFACTS_IMAGE` has a different value in CRW. 

Latest CRW pluginbroker images are https://catalog.redhat.com/software/containers/search?q=pluginbroker&p=1